Automattic/sensei
# Add mail to be sent to the teacher when student sends a new message

Resolves https://github.com/Automattic/sensei/issues/6595

## Proposed Changes

* Added email to be sent to a teacher when a new message is sent from a student to that teacher

## Testing Instructions

- Enable the email post type
- Run the tool Sensei LMS -> Tools -> Re-Create Emails (Do it even if you have done it before)
- Create a course
- Now go to the course as a student and click Contact teacher, send a message (This is the first message)
- Make sure the teacher has received an email

## Pre-Merge Checklist
